contributors/design-proposals/cpu-affinity-numa-awareness.md

```diff
@@ -0,0 +1,216 @@
+**Author**: Derek Carr
+
+**Last** **Updated**: 12/12/2016
+
+**Status**: Pending Review
+
+# CPU Affinity and NUMA Topology Awareness
+
+This proposal describes enhancements to the Kubernetes API to improve the
+utilization of compute resources for containers that have a need to avoid
+cross NUMA node memory access by containers.
+
+This proposal describes the set of changes recommended to support NUMA
+awareness in the `Node` and `Pod` API to support this feature.  It does not
+yet prescribe specifically the actor that performs NUMA aware scheduling
+decisions, instead it focuses purely on the underlying primitives required
+to support building more advanced scheduling capabilities.
+
+## What is NUMA?
+
+Non-uniform memory architecture (NUMA) describes multi-socket machines that
```

Comment:
A in NUMA is "Access" https://en.wikipedia.org/wiki/Non-uniform_memory_access

Should it be "Non-Uniform Memory Access (NUMA) architecture" ?